spec/requests/swagger_spec.rb

```diff
@@ -3337,6 +3339,24 @@
           end
         end
       end
+
+      context 'post request' do
+        it 'returns 201 created status code' do
+          expect(subject).to validate(:post, '/v0/profile/contacts', 201, headers)
+        end
+      end
+
+      context 'patch request' do
+        it 'returns 200 success status code' do
+          expect(subject).to validate(:patch, '/v0/profile/contacts', 200, headers)
+        end
+      end
+
+      context 'delete request' do
+        it 'returns 200 success status code' do
+          expect(subject).to validate(:delete, '/v0/profile/contacts', 200, headers)
+        end
+      end
     end
```

Comment:
Double-check this

Reply:
Based on the example from VA Profile that includes the phrase "sucessfully received" in message text that VHA ES might send, it isn't clear if their 200 should actually be a `201` or a `202: ACCEPTED`…